db/migrate/20231002164646_add_malware_scan_result_to_uploads.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddMalwareScanResultToUploads < ActiveRecord::Migration[7.0]
+  def change
+    add_column :uploads, :malware_scan_result, :string, default: "pending"
```

Comment:
all other enum fields appear to be setup as integer columns, should this be the same?